esp/bindings/SOAP/Platform/soapparam.cpp

```diff
@@ -275,7 +275,7 @@ bool BaseEspStruct::unmarshall(IEspContext* ctx, IProperties &params, MapStrToBu
 void SoapParamBinary::marshall(IRpcMessage &rpc_call, const char *tagname, const char *basepath, const char *xsdtype, const char *prefix)
 {
     StringBuffer sb64;
-    JBASE64_Encode(value.toByteArray(), value.length(), sb64);
+    JBASE64_Encode(value.toByteArray(), value.length(), sb64, true);
```

Comment:
not sure of how this used, are you sure okay to have line breaks here in SoapParam... ?

Reply:
@jakesmith yes I think adding line break is OK here. The value will be base64 decoded before presenting to the user, and the line breaker helps readability.